Dockerfile

```diff
@@ -1,17 +1,10 @@
-FROM node:alpine
+# Dockerfile to build an image of the dpla-frontend application
+#
+# Expect that `yarn build` has already been run.
 
-# Create app directory
-RUN mkdir -p /usr/src/app
-WORKDIR /usr/src/app
-
-# Install app dependencies
-RUN npm install --global yarn
-COPY package.json yarn.lock /usr/src/app/
-RUN yarn
-
-# Bundle app source
-COPY . /usr/src/app
-RUN yarn build
 
+FROM node:carbon-alpine
+WORKDIR /opt/dpla-frontend
+COPY . /opt/dpla-frontend
 EXPOSE 3000
-CMD [ "yarn", "start" ]
\ No newline at end of file
+CMD ["npm", "start"]
```

Comment:
shouldnt we use yarn for everything?

Reply:
It's not necessary for running the application and using npm, which comes for free with Node, means we don't have to jump through any hoops to install Yarn on the Docker image.

Reply:
Yarn is useful for managing dependencies and for its security and efficiency features.